### PR TITLE
BOOKKEEPER-904 test BookieInitializationTest.testDuplicateBookieServerStartup fails on non-english machines

### PR DESCRIPTION
**Pull request by eolivelli**

### CODE REVIEW

File: bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieInitializationTest.java [hunk 1]
```diff
@@ -270,9 +270,7 @@ public void testDuplicateBookieServerStartup() throws Exception {
             fail("Should throw BindException, as the bk server is already running!");
         } catch (ChannelException ce) {
             Assert.assertTrue("Should be caused by a bind exception",
-                              ce.getCause() instanceof BindException);
-            Assert.assertTrue("BKServer allowed duplicate startups!",
```
Comment (fpj) [inline bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieInitializationTest.java]: What's the issue here precisely, @eolivelli?

Comment (eolivelli) [inline bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieInitializationTest.java]: In italian locale  the error is like "indirizzo già in uso" and the test fails. This is annoying on every machine I have and my colleagues

Comment (fpj) [inline bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieInitializationTest.java]: I understand this is annoying, but unfortunately, the change you propose removes an assertion and changes the behavior of the test.

Comment (fpj) [inline bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieInitializationTest.java]: On a second thought, I think this is fine, though. It is sufficient to check for `BindException`.

**Comment by fpj** [comment]
+1, LGTM.

**Pull request closed without merging**